packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -19,6 +19,12 @@
 
 namespace {
 
+#ifdef TV_PROFILE
+constexpr double kProfileFactor = 1.6;
+#else
+constexpr double kProfileFactor = 1.0;
+#endif
```

Comment:
At the current stage, using these constants is fine to me. 
but IMO, this should be passed by the embedder to the platform view.

Reply:
FYI) The terms "scale factor" and "device pixel ratio" are interchangeable, but "profile factor" is not. The profile factor is predefined as a constant for each profile.

https://docs.tizen.org/application/native/guides/ui/efl/multiple-screens/#calculating-the-base-scale